mkosi

```diff
@@ -3145,17 +3151,63 @@ def run_shell(args):
 
         sys.exit(255)
 
+def run_qemu(args):
+
+    if platform.machine() == "x86_64":
+        arch_binary = "qemu-system-x86_64"
+    else:
+        arch_binary = None
+
+    if arch_binary is not None:
+        found = shutil.which(arch_binary)
+    else:
+        found = None
+
+    if found is None:
+        found = shutil.which("qemu")
+
+    if found is None:
+        found = shutil.which("qemu-kvm")
+        if found is None:
+            die("Couldn't find QEMU/KVM binary")
+
+        cmdline = [ found ]
+    else:
+        cmdline = [ found, "-machine", "accel=kvm" ]
+
+    found = None
+    for i in ( '/usr/share/edk2/ovmf/OVMF_CODE.fd', ):
```

Comment:
A more pythonic way to do this would be:

```python
paths = ('/usr/share/edk2/ovmf/OVMF_CODE.fd',)
found = next((p for p in paths if os.path.exists(p)), None)
```

Reply:
Please write this as:
```python
for path in ('/usr/share/edk2/ovmf/OVMF_CODE.fd',):
    if os.path.exists(path):
        break
else:
     die("Couldn't find OVMF UEFI firmware blob.")
```